### PR TITLE
Couple of fixes

### PR DESCRIPTION
**Pull request by thegrb93**

### CODE REVIEW

File: lua/autorun/lua_reloadent.lua [hunk 1]
```diff
@@ -64,9 +64,9 @@
 --   1.0   - First public release                            --
 ---------------------------------------------------------------
 
-AddCSLuaFile("lua_reloadent.lua")
-local FindInLua = VERSION < 150 and file.FindInLua or function(path)
-	return file.Find(path, LUA_PATH)
+AddCSLuaFile()
+local FindInLua = function(path)
+    return file.Find(path, "LUA")
```
Comment (TomyLobo) [inline lua/autorun/lua_reloadent.lua]: You changed tabs to spaces there, can you change that back to tabs and `--amend` your commend?

File: lua/autorun/lua_reloadent.lua [hunk 2]
```diff
@@ -221,7 +221,7 @@ if SERVER then
 	---------------------- Register server-side lua_reloadent ----------------------
 	-- The client is supposed to register less ugly proxy commands for these, which support auto-completition
 	concommand.Add("_lua_reloadentity_sv", function(ply,command,args)
-		if not ply:IsSuperAdmin() then return end
+		if ply:IsValid() and not ply:IsSuperAdmin() then return end
```
Comment (TomyLobo) [inline lua/autorun/lua_reloadent.lua:224]: If the player is invalid, whoever issued the command is automatically considered superadmin.
I assume that's the dedicated server console?

**Comment by thegrb93** [comment]
dedicated server console gives attempt to call method 'IsSuperAdmin' (a nil value). IsValid will return false though.

**Comment by TomyLobo** [comment]
Let me rephrase my question:
Is there any non-privileged way of issuing a command without a valid player?

**Comment by thegrb93** [comment]
No, player is only invalid if it was issued with server console. I asked the forums about that too a while ago to be sure. I'll add that to the wiki as well.

**Comment by TomyLobo** [comment]
Alright, that's what I wanted to know
